### PR TITLE
add apptainer.registry

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by ewels — APPROVED** [review]
LGTM, thanks for the speedy PR!

**Review by ewels — CHANGES_REQUESTED** [review]
Needs changelog

**Review by mirpedrol — APPROVED** [review]
thanks!

**Comment by maxulysse** [comment]
And we have a CHANGELOG even

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - Fix parsing of container directive when it is not typical nf-core format ([#2306](https://github.com/nf-core/tools/pull/2306))
 - Add ability to specify custom registry for linting modules, defaults to quay.io ([#2313](https://github.com/nf-core/tools/pull/2313))
 - Add `singularity.registry = 'quay.io'` in pipeline template ([#2305](https://github.com/nf-core/tools/pull/2305))
+- Add `apptainer.registry = 'quay.io'` in pipeline template ([#2305](https://github.com/nf-core/tools/pull/2352))
```
Comment (mirpedrol) [inline CHANGELOG.md:20]:
```suggestion
- Add `apptainer.registry = 'quay.io'` in pipeline template ([#2352](https://github.com/nf-core/tools/pull/2352))
```

**Pull request merged**